gst/gst_message.go

```diff
@@ -268,7 +268,7 @@ type StepDoneValues struct {
 	Rate         float64
 	Flush        bool
 	Intermediate bool
-	Duration     time.Duration
+	Duration     time.Duration // should this also be a clocktime?
```

Comment:
No, its not a ClockTime, we should _try_ to make things 1:1 as much as possible, just because something can be set to `GST_CLOCK_TIME_NONE` doesn't mean it should be a ClockTime